discord/commands/context.py

```diff
@@ -220,14 +220,15 @@ def send_modal(self) -> Callable[..., Awaitable[Interaction]]:
         """Sends a modal dialog to the user who invoked the interaction."""
         return self.interaction.response.send_modal
 
-    @property
-    def respond(self) -> Callable[..., Awaitable[Union[Interaction, WebhookMessage]]]:
-        """Callable[..., Union[:class:`~.Interaction`, :class:`~.Webhook`]]: Sends either a response
-        or a followup response depending on if the interaction has been responded to yet or not."""
-        if not self.interaction.response.is_done():
-            return self.interaction.response.send_message  # self.response
-        else:
-            return self.followup.send  # self.send_followup
+    async def respond(self) -> Union[Interaction, WebhookMessage]:
```

Comment:
```suggestion
    async def respond(self, *args, **kwargs) -> Union[Interaction, WebhookMessage]:
```
Need to include *args and **kwargs here.